.github/labeler.yml

```
@@ -0,0 +1,193 @@
+# numpy-stubs module
+
+numpy.char.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/char/**
+
+numpy._core.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/_core/**
+
+numpy.core.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/core/**
+
+numpy.ctypeslib.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/ctypeslib.pyi
+
+numpy.distutils.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/distutils/**
+
+numpy.dtype:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/core/numerictypes.pyi
+    - src/numpy-stubs/dtypes.pyi
+
+numpy.dtypes.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/dtypes.pyi
+
+numpy.emath.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/lib/emath.pyi
```

confusingly, this module is actually called `scimath` for some reason 🤷🏻 
```suggestion
    - src/numpy-stubs/lib/_scimath_impl.pyi
    - src/numpy-stubs/lib/scimath.pyi
```